tt_metal/impl/dispatch/device_command.hpp

```diff
@@ -45,6 +45,7 @@ struct CommandHeader {
     uint32_t restart = 0;
     uint32_t new_issue_queue_size = 0;
     uint32_t new_completion_queue_size = 0;
+    uint32_t fwd_path = 1;
```

Comment:
this is kind of ugly but added this so the routers on the fwd and bwd path know when to expect data being tunnelled through versus when they should drop data

the remote cmd signaller unsets this so the src/dst routers on the bwd path know that data is coming back with a read